src/Umbraco.Web.UI.Client/src/less/components/umb-search-filter.less

```diff
@@ -0,0 +1,28 @@
+html .umb-search-filter {
```

Comment:
@BatJan is there a specific need to include `html` here? Unless `umb-search-filter` can be a class on `html` element itself?

Reply:
@bjarnef It's due to specificity. I would like to avoid using `!important` and adding html as the first selector makes that possible and provides the needed specificity. It's been a long while and I have not touched any code for a little over 2 months but I think I faced an issue where the declaration was too generic changing the appearance elsewhere. Maybe in a scenario where the class was used some place where it might not have belonged... Can't really remember it now 🤷 😄

Reply:
@BatJan it just seemed a bit unnecessary and the `w-100` which already exists as a utility class. However I tried removing this, but then the `forms.less` overwritten to padding on the input styling.
Anyway I fixed some issues after the `icon-search` class was removed from `<umb-icon>` in another PR.
https://github.com/umbraco/Umbraco-CMS/pull/9631